internal/resolve/catalog.go

```diff
@@ -28,11 +31,17 @@ type CatalogResolver struct {
 	Validations      []ValidationFunc
 }
 
+type foundBundle struct {
+	bundle   *declcfg.Bundle
+	catalog  string
+	priority int32
+}
```

Comment:
Any reason this type needs to be used outside the `Resolve` function? If not, it seems better IMO to leave it scoped to that function.

Reply:
I use it as a field type in the `resolutionErrorInfo` struct